python/pdo/service_client/enclave.py

```diff
@@ -48,25 +58,38 @@ def encryption_key(self) :
     def enclave_id(self) :
         return self.enclave_keys.identity
 
+    # -------------------------------------------------------
+    def _attach_storage_service_(self, storage_service) :
+        self.storage_service_verifying_key = storage_service.verifying_key
+
+        self.get_block = storage_service.get_block
+        self.get_blocks = storage_service.get_blocks
+        self.store_block = storage_service.store_block
+        self.store_blocks = storage_service.store_blocks
+        self.check_block = storage_service.check_block
+        self.check_blocks = storage_service.check_blocks
+
     # -----------------------------------------------------------------
     # encrypted_session_key -- base64 aes key encrypted with enclave's rsa key
     # encrypted_request -- base64 string encrypted with aes session key
     # -----------------------------------------------------------------
     def send_to_contract(self, encrypted_session_key, encrypted_request) :
-        request = { 'operation' : 'UpdateContractRequest' }
+        request = dict()
         request['encrypted_session_key'] = encrypted_session_key
         request['encrypted_request'] = encrypted_request
 
         try :
-            response = self._postmsg(request)
-            return response['result']
+            url = '{0}/invoke'.format(self.ServiceURL)
```

Comment:
I guess you are using requests instead of urllib for send_to_contract even in PR152